unit-tests/unit-tests-common.h

```diff
@@ -317,64 +317,10 @@ inline bool file_exists(const std::string& filename)
 inline bool make_context(const char* id, rs2::context* ctx, std::string min_api_version = "0.0.0")
 {
     rs2::log_to_file(RS2_LOG_SEVERITY_DEBUG);
+    min_api_version;
 
-    static std::map<std::string, int> _counters;
-
-    _counters[id]++;
-
-    auto argc = command_line_params::instance().get_argc();
-    auto argv = command_line_params::instance().get_argv();
-
-
-    std::string base_filename;
-    bool record = false;
-    bool playback = false;
-    for (auto i = 0u; i < argc; i++)
-    {
-        std::string param(argv[i]);
-        if (param == "into")
-        {
-            i++;
-            if (i < argc)
-            {
-                base_filename = argv[i];
-                record = true;
-            }
-        }
-        else if (param == "from")
-        {
-            i++;
-            if (i < argc)
-            {
-                base_filename = argv[i];
-                playback = true;
-            }
-        }
-    }
-
-    std::stringstream ss;
-    ss << id << "." << _counters[id] << ".test";
-    auto section = ss.str();
-
-
-    try
-    {
-        if (record)
-        {
-            *ctx = rs2::recording_context(base_filename, section);
-        }
-        else if (playback)
-        {
-            *ctx = rs2::mock_context(base_filename, section, min_api_version);
-        }
-        command_line_params::instance()._found_any_section = true;
-        return true;
-    }
-    catch (...)
-    {
-        return false;
-    }
-
+    command_line_params::instance()._found_any_section = true;
+    return true;
```

Comment:
Should this method be kept if this is the only things it does?

Reply:
It's also modified, but differently, on the `dds` branch. Live tests are old and we'll not invest much in improving them. For now this change you're suggesting may happen, but in a different PR.

Reply:
ok, please open a ticket for that

Reply:
No, Remi. We won't waste time on live-tests. I'll do it if I happen to be messing around in live-tests again. Leave it be... you're welcome to open a ticket if you insist.

Reply:
One thing I can offer is that right after we merge this I'll be merging with the dds branch, where there will be conflicts. I'll take a look then to see what can be removed.

Reply:
ok